### DISCUSSION TITLE
About table's export

### DISCUSSION THREAD

**Discussion by dlyltm** [BlazorComponent/MASA.Blazor#1572] [Q&A]
How to export the contents in < MDataTable > to excel table?

**Comment by capdiem** [comment]
NPOI or other Excel class libraries, the DataTable component will not support this feature.

**Comment by VleaStwo** [comment]
`MDataTable`只是一个UI显示

本身的 `Items` 才是要导出的内容

而 `Items` 控制权在你手上

你可以遍历 `Items` 输出

如果你要连结 `MDataTable` 的 `Header` 的显示顺序 也完全可以从 数据角度的`Header`入手

可以对你的 `TItem` 增加一个实例方法 输出一个 可以传递给如 **NPOI** 等库可用的数据  (对于table来说 是一个row)
